docs/content/preview/benchmark/resilience/jepsen-testing.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Jepsen testing
+headerTitle: Jepsen testing
+linkTitle: Jepsen testing
+description: Description of Jepsen testing.
+menu:
+  preview:
+    identifier: jepsen-testing-1
+    parent: resilience
+    weight: 13
+aliases:
+  - /preview/benchmark/resilience/jepsen-testing-ysql/
+  - /preview/benchmark/resilience/jepsen-testing-ycql/
+type: docs
+---
+
+
+This documentation page contains descriptions of Jepsen tests that YugabyteDB runs daily for each
```

Comment:
```suggestion
This page describes Jepsen tests that YugabyteDB runs daily for each major releases currently available.
```

Reply:
`This page describes Jepsen tests that YugabyteDB runs daily for each major release currently available.`
Changed to suggested line, except `release` word in singular form